async_processor/app.py

```diff
@@ -105,9 +100,17 @@ def _ready_route_handler(self):
     def _healthy_route_handler(self):
         return ""
 
-    def _process_route_handler(self, body: InputMessage) -> Response:
+    async def _process_route_handler(
+        self,
+        request: Request,
+        # We need this ` _: dict = Body(...)` to
+        # ensure that the user can put request body in the Swagger page
+        _: dict = Body(...),  # noqa: B008
+    ) -> Response:
         start = time.perf_counter()
-        output = self._processor.process(body)
+        payload = await request.body()
+        input_message = self._processor.input_deserializer(payload)
```

Comment:
Does `request.body()` also do some decoding and `input_deserializer` expects bytes?

Reply:
`input_deserializer` can work with both strings and bytes.